Formula/newsboat.rb

```diff
@@ -0,0 +1,26 @@
+class Newsboat < Formula
+  desc "RSS/Atom feed reader for text terminals"
+  homepage "https://newsboat.org/"
+  url "https://github.com/newsboat/newsboat/archive/r2.10.1.tar.gz"
+  sha256 "82d5e3e2a6dab845aac0bf72580f46c2756375d49214905a627284e5bc32e327"
+  head "https://github.com/newsboat/newsboat.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "json-c"
+  depends_on "libstfl"
+
+  needs :cxx11
+
+  def install
+    ENV.cxx11
+    system "make"
+    system "make", "install", "prefix=#{prefix}"
+  end
+
+  test do
+    urlfile = "urls.txt"
+    (testpath/urlfile).write "https://github.com/blog/subscribe\n"
+    assert_match /newsboat - Exported Feeds/m, shell_output("#{bin}/newsboat -e -u #{urlfile}")
```

Comment:
```
assert_match /newsboat - Exported Feeds/m, shell_output("#{bin}/newsboat -e -u urls.txt")
```

Reply:
Doesn't work. It doesn't seem `newsboat` gets invocated from `testpath`

Reply:
In fact, it's just that `newsboat` doesn't start without locale information